python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -29,8 +29,24 @@ router bgp {{ router_bgp.as }}
 {%     elif router_bgp.bgp.default.ipv4_unicast_transport_ipv6 is arista.avd.defined(false) %}
    no bgp default ipv4-unicast transport ipv6
 {%     endif %}
-{%     if router_bgp.timers is arista.avd.defined %}
-   timers bgp {{ router_bgp.timers }}
+{%     if router_bgp.timers.keepalive_time is arista.avd.defined and
+        (router_bgp.timers.hold_time is arista.avd.defined or router_bgp.timers.hold_forever is arista.avd.defined) %}
+{%         set timers_bgp_cli = "timers bgp " ~ router_bgp.timers.keepalive_time %}
+{%         if router_bgp.timers.hold_time is arista.avd.defined %}
+{%             set timers_bgp_cli = timers_bgp_cli ~ " " ~ router_bgp.timers.hold_time %}
+{%         else %}
+{%             set timers_bgp_cli = timers_bgp_cli ~ " 0" %}
+{%         endif %}
```

Comment:
Adding the 0 here seems wrong. Is that how EOS behaves? If so change it to just add `router_bgp_timers.hold_time | arista.avd.default(0)` and update the default value in the schema.

Reply:
0 is for hold_forever

Reply:
you meant this?
```
{%     if router_bgp.timers.hold_forever is arista.avd.defined(true) %}
{%        set timers_bgp_cli = " timers bgp 0 0" %}
   {{ timers_bgp_cli }}
{%     elif router_bgp.timers.keepalive_time is arista.avd.defined and router_bgp.timers.hold_time is arista.avd.defined %}
{%         set timers_bgp_cli = "timers bgp " ~ router_bgp.timers.keepalive_time " " ~ router_bgp.timers.hold_time %}
   {{ timers_bgp_cli }}
```

Reply:
if `hold_forever` is just an alias for `timers bgp 0 0` we should not implement it at all in our schema. Only implement things that are in the running config.

Reply:
Updated the schema